sys/dev/ath/if_ath_pci.c

```diff
@@ -279,7 +281,7 @@ ath_pci_attach(device_t dev)
 	}
 #endif /* ATH_EEPROM_FIRMWARE */
 
-	error = ath_attach(pci_get_device(dev), sc);
+	error = ath_attach(ATHEROS_VENDOR_ID, pci_get_device(dev), sc);
```

Comment:
Shouldn't it be: `pci_get_vendor(dev)`

Reply:
It's just moved from sys/dev/ath/ath_hal/ah.c file.
`if (chip->probe(ATHEROS_VENDOR_ID, devid) == AH_NULL) `
I don't know why there is fixed value and if we can use `pci_get_vendor(dev)`.